features/frameworks/COMPONENT_FPGA_CI_TEST_SHIELD/source/MbedTester.cpp

```diff
@@ -26,23 +26,25 @@
 
 #define mbed_tester_printf(...)
 
-#define PHYSICAL_PINS           128
-#define LOGICAL_PINS            8
-#define FIRMWARE_SIZE           2192012
-#define FIRMWARE_REGION_SIZE    0x220000
-#define FIRMWARE_HEADER_SIZE    0x10000
-#define FLASH_SECTOR_SIZE       0x1000
-#define LENGTH_SIZE             0x4
-#define CRC_SIZE                0x4
-#define FLASH_SPI_FREQ_HZ       2000000
-#define ANALOG_COUNT            4
-
-#define PHYSICAL_NC    ((MbedTester::PhysicalIndex)0xFF)
-
-static const uint8_t KEY[8] = {
-    0x92, 0x9d, 0x9a, 0x9b,
-    0x29, 0x35, 0xa2, 0x65
-};
+namespace {
+    const auto physical_pins = 128;
+    const auto logical_pins = 8;
+    const auto firmware_size_const = 2192012;
+    const auto firmware_region_size = 0x220000;
+    const auto firmware_header_size = 0x10000;
+    const auto flash_sector_size = 0x1000;
+    const auto length_size = 0x4;
+    const auto crc_size_const = 0x4;
+    const auto flash_spi_freq_hz = 2000000;
+    const auto analog_count = 4;
```

Comment:
some variables end with `_const` some others don't.

I think UPPERCASE makes it more obvious that those are constants.

```suggestion
    const auto FIRMWARE_SIZE        = 2192012;
    const auto FIRMWARE_REGION_SIZE = 0x220000;
    const auto FIRMWARE_HEADER_SIZE = 0x10000;
    const auto FLASH_SECTOR_SIZE    = 0x1000;
    const auto LENGTH_SIZE          = 0x4;
    const auto CRC_SIZE             = 0x4;
    const auto FLASH_SPI_FREQ_HZ    = 2000000;
    const auto ANALOG_COUNT         = 4;
```

Reply:
There are variables with internal linkage, uppercase made it possible to clash with the preprocessor and would rather indicate they are macros and with external linkage - a reason this PR was proposed is to avoid this.

`_const` shoud be removed

Reply:
what about the `k` prefix? or use a named namespace `namespace MbedTesterConst {`

Reply:
This file has other sort of inconsistencies. Around line 460 lies global static const arrays in full CAPS.

Moving them all to an `MbedTesterConst` namespace seems sensible to me.
Lowercase name to avoid clash with preprocessor in a namespace explicit about what those values are is IMHO a good compromise.

I would also recommend to move the magic value for PhysicalIndex to the header file as this is where the type is publicly visible.